test-framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -161,6 +161,25 @@ public static void restoreContentType() {
         Requests.INDEX_CONTENT_TYPE = XContentType.JSON;
     }
 
+    private static boolean TESTS_SEED_PROPERTY_SET = false;
+    @BeforeClass
+    @SuppressForbidden(reason = "sets tests.seed")
```

Comment:
@rmuir Thanks for the insightful explanations. I've removed this logic in a5d96069052cc2304056115aa1af50e1e199bef7 (with a detailed commit message for future weary travelers that try to fix IntelliJ JUnit integration).